### DISCUSSION TITLE
Hosting RabbitMQ on Azure

### DISCUSSION THREAD

**Discussion by Anu666** [rabbitmq/rabbitmq-server#2464] [Questions] [answered]
I need to read and publish messages to a Rabbit MQ instance from multiple app services on Azure.

Could anyone please suggest the Azure service that I should be using to host the Rabbit MQ instance?

**Comment by michaelklishin** [comment, marked as answer]
I am not aware of a RabbitMQ-as-a-Service option on Azure but CloudAMQP lists it as a supported IaaS provider. You can always provision it using one of the [existing deployment options that are not IaaS-specific](https://www.rabbitmq.com/download.html), e.g. using Docker, [Kubernetes](https://www.rabbitmq.com/kubernetes/operator/operator-overview.html), Chef, Terraform RabbitMQ provider, and so on.
